src/main/features/claim/views/defendant-represented.njk

```diff
@@ -7,7 +7,12 @@
 {% block content %}
 
   <h1 class="heading-large">
-    {% if defendantNumber %} <span class="form-hint">{{ defendantNumber }} </span> {% endif %}
+    <span class="form-hint">
+      {% if defendantNumber %} {{ defendantNumber }}
```

Comment:
Does `defendantNumber` also contains the name to display? If yes, then probably we should name it accordingly. Otherwise we are missing name to display.